lib/upload.js

```diff
@@ -479,6 +479,11 @@ class Upload {
 
         this._emitProgress(start + e.loaded, this._size);
       };
+
+      xhr.upload.onloadstart = (e) => {
+        // Emit an progress event when a new chunk begins being uploaded.
+        this._emitProgress(start, this._size);
+      };
```

Comment:
This block is not needed anymore. tus-js-client already emits a progress even if it start uploading a chunk (see https://github.com/tus/tus-js-client/blob/master/lib/upload.js#L502).